src/libraries/System.Net.Requests/ref/System.Net.Requests.cs

```diff
@@ -486,6 +486,7 @@ public partial class ServicePoint
         public bool CloseConnectionGroup(string connectionGroupName) { throw null; }
         public void SetTcpKeepAlive(bool enabled, int keepAliveTime, int keepAliveInterval) { }
     }
+    [System.ObsoleteAttribute("WebRequest, HttpWebRequest, ServicePoint, and WebClient are obsolete. Use HttpClient instead.", DiagnosticId = "SYSLIB0014", UrlFormat = "https://aka.ms/dotnet-warnings/{0}")]
```

Comment:
I would like it to be mentioned in the message that ServicePointManager members do not affect SslStream and HttpClient behavior (which users mistakenly keep assuming after migrating from .NET Framework).

Reply:
Added a simple `Settings on ServicePointManager no longer affect SslStream or HttpClient.` suffix to the message (just on ServicePointManager).